common/messages/messages_en.go

```diff
@@ -170,6 +170,62 @@ var messagesEn = map[string]util.Message{
 		RC:          500,
 		Action:      "Wait for file share deletion",
 	},
+	"ListSubnetsFailed": {
+		Code:        "ListSubnetsFailed",
+		Description: "Unable to fetch list of subnet.",
+		Type:        util.RetrivalFailed,
+		RC:          500,
+		Action:      "Unable to list subnet. Run 'ibmcloud is subnets' to list available subnets in your account.",
+	},
+	"NextSubnetPageParsingError": {
+		Code:        "NextSubnetPageParsingError",
+		Description: "The next field '%s' specified in the next parameter of the list subnet call could not be parsed.",
+		Type:        util.RetrivalFailed,
+		RC:          500,
+		Action:      "Please verify that the next field is correct.",
+	},
+	"StartSubnetIDEmpty": {
+		Code:        "StartSubnetIDEmpty",
+		Description: "The start '%s' specified in the next parameter of the list subnet call is empty.",
+		Type:        util.RetrivalFailed,
+		RC:          500,
+		Action:      "Please verify that the start field is correct.",
+	},
+	"SubnetFindFailedWithZoneAndSubnetID": {
+		Code:        "SubnetFindFailedWithZoneAndSubnetID",
+		Description: "A subnet with the specified zone '%s' and available cluster subnet list '%s' could not be found.",
+		Type:        util.RetrivalFailed,
+		RC:          404,
+		Action:      "Verify that the subnet from the cluster subnet list exists. Run 'ibmcloud is subnets' to list available subnets in your account.",
+	},
+	"ListSecurityGroupsFailed": {
+		Code:        "ListSecurityGroupsFailed",
+		Description: "Unable to fetch list of securityGroup.",
+		Type:        util.RetrivalFailed,
+		RC:          500,
+		Action:      "Unable to list securityGroup. Run 'ibmcloud is securityGroups' to list available securityGroups in your account.",
+	},
+	"NextSecurityGroupPageParsingError": {
+		Code:        "NextSecurityGroupPageParsingError",
+		Description: "The next field '%s' specified in the next parameter of the list securityGroup call could not be parsed.",
```

Comment:
This is not helpful for user as user can't understand this and use can't do any thing. Better to provide an error which is useful for user and what user need to do to fix the issue